taf/testlib/virtual_env.py

```diff
@@ -248,7 +250,7 @@ def wait_for_server_status(self, vm_id, status):
         from tempest.common import waiters
         return waiters.wait_for_server_status(self.handle.servers_client, vm_id, status)
 
-    def ensure_public_access(self, try_reuse=False, networks_client=None, routers_client=None,
+    def ensure_public_access(self, try_reuse=True, networks_client=None, routers_client=None,
```

Comment:
Real default value here was never "False". Function is called with value taken from command line option --reuse-venv, which has default value "True". I just made it consistent.